repos/fdbt-site/src/pages/returnService.tsx

```diff
@@ -96,6 +101,7 @@ export const getServerSideProps = async (ctx: NextPageContextWithSession): Promi
 
     const ticket = getSessionAttribute(ctx.req, MATCHING_JSON_ATTRIBUTE);
     const matchingJsonMetaData = getSessionAttribute(ctx.req, MATCHING_JSON_META_DATA_ATTRIBUTE);
+    const modesAttribute = getSessionAttribute(ctx.req, MULTI_MODAL_ATTRIBUTE);
     const nocCode = getAndValidateNoc(ctx);
     const selectedServiceId = Number(ctx.query.selectedServiceId);
```

Comment:
Do we not need to change the text on this page to mention TNDS instead of BODS? line 88

Reply:
It should probably also get the dataSource attribute like how it is done in service.tsx line 80.

Reply:
i am using the dataSource as string instead of passing the whole attribute